src/Main.elm

```diff
@@ -112,10 +112,15 @@ stringFromViewMode viewMode =
             "CollectData"
 
 
+type alias System =
+    { name : String, description : String }
+
+
 type Presence
     = Required
     | Optional
-    | System { name : String, description : String }
```

Comment:
we should probably retain the `System` too for backwards compatibility?

Reply:
we can update decoder to achieve backward compatibility https://github.com/choonkeat/tiny-form-fields/pull/1/files#r1729699648

new jsons encoded won’t have `"System"` anymore.